bool find_obs_flag
float32 dis
float32 pos_left
float32 pos_right
float32 x_pos
float32 y_pos